src/dev/impl/DevToys/Views/Tools/EncodersDecoders/JwtDecoderEncoder/JwtDecoderControl.xaml.cs

```diff
@@ -51,5 +65,22 @@ private void PayloadCodeEditor_ExpandedChanged(object sender, System.EventArgs e
                 JwtDecoderGrid.Children.Add(PayloadCodeEditor);
             }
         }
+
+        private void OnLoaded(object sender, RoutedEventArgs e)
+        {
+            ApplySettings();
+        }
+
+        private void ApplySettings()
+        {
+            if (_settingsProvider is not null && InfoDataGrid is not null)
+            {
+                FontFamily fontFamily = new(_settingsProvider.GetSetting(PredefinedSettings.TextEditorFont));
+                InfoDataGrid.FontFamily = fontFamily;
+                Style style = new(typeof(DataGridColumnHeader));
+                style.Setters.Add(new Setter(FontFamilyProperty, fontFamily));
```

Comment:
Can't we use Binding with a DependencyProperty instead?

Reply:
I think I was able to do what you're suggesting in c1a64ccf75b936a72a8b8814d7819bc4d0611da4. I was also able to eliminate the `Loaded` handler, and I didn't need to set the column headers separately.

Reply:
It looks much better! :D